src/main/java/org/elasticsearch/search/aggregations/reducers/movavg/models/MovAvgModelBuilder.java

```diff
@@ -0,0 +1,10 @@
+package org.elasticsearch.search.aggregations.reducers.movavg.models;
+
+import org.elasticsearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+
+public interface MovAvgModelBuilder {
```

Comment:
This should probably have some JavaDocs since it forms part of the Java API. Also, should this not implement toXContent?